docat/docat/utils.py

```diff
@@ -126,7 +126,7 @@ def is_forbidden_project_name(name: str) -> bool:
     a page on the docat website.
     """
     name = name.lower().strip()
-    return name in ["upload", "claim", "delete", "help"]
+    return name in ["upload", "claim", "delete", "help", "doc"]
```

Comment:
This is fine with me @fliiiix what do you think?

Reply:
It seems that `api` should also be added to the list...